.github/pr-custom-review.yml

```diff
@@ -20,15 +20,16 @@ rules:
     condition:
       include: .*
       # excluding files from 'Runtime files' and 'CI files' rules
-      exclude: ^runtime/(kusama|polkadot)/src/[^/]+\.rs$|^\.gitlab-ci\.yml|^scripts/ci/.*|^\.github/.*
+      exclude: ^runtime/(kusama|polkadot)/src/[^/]+\.rs$|^\.gitlab-ci\.yml|^(?!.*\.dic$|.*spellcheck\.toml$)scripts/ci/.*|^\.github/.*
     min_approvals: 2
     teams:
       - core-devs
 
   - name: CI files
     check_type: changed_files
     condition:
-      include: ^\.gitlab-ci\.yml|^scripts/ci/.*|^\.github/.*
+      # dictionary files are excluded
+      include: ^\.gitlab-ci\.yml|^(?!.*\.dic$|.*spellcheck\.toml$)scripts/ci/.*|^\.github/.*
```

Comment:
Should this be `^scripts/ci/(?!.*\.dic$|.*spellcheck\.toml)`? As in: match all files within the `scripts/ci/` directory, except ones that end up with `.dic` or `spellcheck.toml`.

 I'm not sure if this work as-is but I guess it does? I don't know if you've tested it somehow